pkg/plugins/runtime/k8s/controllers/pod_controller.go

```diff
@@ -105,6 +105,16 @@ func (r *PodReconciler) Reconcile(ctx context.Context, req kube_ctrl.Request) (k
 }
 
 func (r *PodReconciler) reconcileDataplane(ctx context.Context, pod *kube_core.Pod, log logr.Logger) error {
+	ns := kube_core.Namespace{}
+	if err := r.Client.Get(ctx, kube_types.NamespacedName{Name: pod.Namespace}, &ns); err != nil {
+		return errors.Wrap(err, "unable to get Namespace for Pod")
```

Comment:
Should you ignore namespace not found here as well?

Reply:
I'm not sure we can get there when the namespace is deleted. I definitely haven't noticed this when I was playing with this namespace termination